contracts/core/RebalancingSetToken.sol

```diff
@@ -255,7 +254,7 @@ contract RebalancingSetToken is
         require(block.timestamp >= proposalStartTime.add(proposalPeriod), "PROPOSAL_PERIOD_NOT_ELAPSED");
 
         // Get core address from factory and create core interface
-        address coreAddress = ISetFactory(factory).core();
+        address coreAddress = IRebalancingSetFactory(factory).core();
         ICore core = ICore(coreAddress);
```

Comment:
Advise changing these 2 lines to `ICore core = ICore(IRebalancingSetFactory(factory).core());` to mirror line 192.

Reply:
+1